src/components/Main.tsx

```diff
@@ -0,0 +1,87 @@
+import cn from 'classnames';
+import { useState } from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  onTodos: Todo[]
+  onDeleteTodo: (value: number) => void
+  onUpdateTodo: (value: Todo) => void
+};
+
+export const Main: React.FC<Props>
+
+  = ({ onTodos, onDeleteTodo, onUpdateTodo }) => {
+    const [isEditing] = useState(false);
+
+    return (
+      <section className="todoapp__main">
+        {onTodos.map(todo => (
+          <>
```

Comment:
you `.map` in JSX - mapped elements must have some mandatory attribute...
remind how you can add this attribute to react fragment.